src/features/ReactToPost/AnimatedCard.tsx

```diff
@@ -0,0 +1,507 @@
+                        selected={internalReaction === emojiRaisedHands}
+                        onPress={() => setSelected(emojiRaisedHands)}
+                      />
+                    </Animated.View>
+                    <Animated.View
+                      style={[styles.emojiContainer, tearSmileAnimatedStyles]}
+                      onLayout={event => {
+                        emojiContainerPositions.value = {
+                          ...emojiContainerPositions.value,
+                          tearSmile: event.nativeEvent.layout.x,
+                        };
+                      }}>
+                      <EmojiButton
+                        key={emojiTearSmile}
+                        position={panPosition.value}
+                        emoji={emojiTearSmile}
+                        selected={internalReaction === emojiTearSmile}
+                        onPress={() => setSelected(emojiTearSmile)}
+                      />
+                    </Animated.View>
+                    <Animated.View
+                      style={[styles.emojiContainer, confusedAnimatedStyles]}
+                      onLayout={event => {
+                        emojiContainerPositions.value = {
+                          ...emojiContainerPositions.value,
+                          confused: event.nativeEvent.layout.x,
+                        };
+                      }}>
+                      <EmojiButton
+                        key={emojiConfused}
+                        position={panPosition.value}
+                        emoji={emojiConfused}
+                        selected={internalReaction === emojiConfused}
+                        onPress={() => setSelected(emojiConfused)}
+                      />
+                    </Animated.View>
+                    <Animated.View
+                      style={[styles.emojiContainer, laughAnimatedStyles]}
+                      onLayout={event => {
+                        emojiContainerPositions.value = {
+                          ...emojiContainerPositions.value,
+                          laugh: event.nativeEvent.layout.x,
+                        };
+                      }}>
+                      <EmojiButton
+                        key={emojiLaugh}
+                        position={panPosition.value}
+                        emoji={emojiLaugh}
+                        selected={internalReaction === emojiLaugh}
+                        onPress={() => setSelected(emojiLaugh)}
+                      />
+                    </Animated.View>
+                  </View>
+                </Animated.View>
+              </Portal>
+            </View>
+          </View>
+        </View>
+      </View>
+    </GestureDetector>
+  );
+}
+
+const EmojiButton = (props: {
+  emoji: string;
+  selected: boolean;
+  position: number;
+  onPress: () => Promise<void> | void;
+}) => {
+  return (
+    <Pressable onPress={props.onPress}>
+      <Text style={[{fontSize: 26}]}>{props.emoji}</Text>
```

Comment:
```suggestion
      <Text style={{fontSize: 26}}>{props.emoji}</Text>
```
no precisas los corchetes rectos ya que es un solo objeto

Reply:
Lo muevo a la constante de estilos.